### ISSUE TITLE
Use local validator for test-example-validation.js

### ISSUE THREAD

**Issue by aghassemi** [ampproject/amphtml#9002]
from `test-example-validation.js`:
```
// TODO(@cramforce): Use local version. This is non-hermetic
// and really bad. When the validator is open source we can
// use it directly.
```

validator is now open-source so we can use the local version.

Ran into this with: https://github.com/ampproject/amphtml/pull/8632#discussion_r113650888 

Not sure 
/cc @erwinmombay @rsimha-amp @honeybadgerdontcare

**Comment by rsimha-amp** [comment]
Assigning to @choumx, who is working on #10158 to fix this issue and #10154

**Comment by choumx** [comment]
Oops, didn't know this existed. 😄  Closing in favor of #10154.
